linux/deploy_vm/deploy_vm_from_iso.yml

```diff
@@ -201,6 +201,19 @@
   vars:
     vm_take_screenshot_local_path: "{{ current_test_log_folder }}"
 
+# VM might not have MAC address after creation without power-on
```

Comment:
"without power-on"? VM is powered on in line 131.

Reply:
When the VM's MAC address is allocated by ESXi host, it only gets the MAC after VM is powered on. In this test case, when the first time of executing vm_wait_primary_nic_mac.yml, the VM is powered off, so its mac is null until it is powered on.